index.html

```diff
@@ -1,19 +1,104 @@
+            </header>
+            <!--main-->
+             <main class="main">
+                 <div class="intro">
+                    <h1>Introducing Karma</h1>
+                    <h2>Bring WiFi with you, everywhere you go.</h2>
+                    <button>Learn More</button>
+                 </div>
+                 <!--Everyone Needs a little Karma-->
+                 <div class="everyone">
+                        <h3>Everyone needs a little Karma.</h3>
+                        <!--tree items container-->
+                        <div class="tree_item">
+                            <div class="item">
+                                <img src="img/icon-devices.svg" alt="devices"class="icon">
+                                <p> Internet for all devices</p>
+                            </div>
+                            <div class="item">
+                                <img src="img/icon-coffee.svg" alt="coffee"class="icon">
+                                <p>Boost your productivity</p>
+                            </div>
+                            <div class="item">
+                                <img src="img/icon-refill.svg" alt="refill"class="icon">
+                                <p>Pay as You Go</p>
+                            </div>
+                        </div>
+                 </div>
+                 <div class="extention-container">
+                    <div class="extention-image">
+                    <img src="./additional/feature-img.png">
+                    </div>
+                    <div class="extention-image1">
+                            <p>
+                                Wherever I am, I just don't <br>worry about my connection <br>anymore!
+                            </p>
+                            <button><a href="store.html">Get karma today</a></button>
+                    </div>
+                 </div>
+                </main>
+                 <!--fotter-->
+                 <footer class="footer">
+                     <p>Join us on</p>
+                     <div class="for_join">
+                    <a href="#">
+                        <img src="img/twitter-icon.svg" alt="twitter" class="social_media">
+                    </a>
+                    <a>
+                        <img src="img/facebook-icon.svg" alt="facebook" class="social_media">
+                    </a>
+                    <a>
+                        <img src="img/instagram-icon.svg" alt="instagram" class="social_media">
+                    </a>
+                    
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+                     </div>
+                        <p>&#169; Karma Mobilty, Inc</p>
+                 </footer>
 
-</body>
+
+             
+        
+             
+    </body>
 </html>
+   <!-- Add your HTML markup here -->
```

Comment:
It is a good practice to remove "dead" or unused code. It doen't look this comments provide much value to the reader of the code - can be safely removed.

Reply:
Yes, sure.